### PR TITLE
Remove minKotsVersion limitations

### PR DESCRIPTION
**Pull request by ajp-io**

### CODE REVIEW

File: content/reference/v1beta1/application.md [hunk 1]
```diff
@@ -184,19 +184,13 @@ The minimum KOTS version that is required by the release. Introduced in KOTS v1.
...
-* The minimum version increases and then decreases from one release to the next.
+### Guidance for informing users of the need to update
 
-`minKotsVersion` is not supported in these cases because when the minimum version increases, the admin console can error when retrieving it (described [below](#guidance-for-informing-users-of-the-need-to-update)). If the minimum KOTS version then decreases in the next release, KOTS will successfully retrieve that release, which means the intermediate release is lost.
+After promoting a new release that specifies a minimum KOTS version that is greater than what an end user currently has deployed, that application version appears in the version history of the admin console after checking for updates, but the application version is not downloaded. The admin console temporarily displays an error message inline with this pending version, informing the end user that they must update KOTS in order to download this application version. This error is also displayed when the end user checks for updates with the [`kots upstream upgrade` command](/kots-cli/upstream/).
```
Comment (sgalsaleh) [inline content/reference/v1beta1/application.md:189]:
```suggestion
After promoting a new release that specifies a minimum KOTS version that is greater than what an end user currently has deployed, that application version appears in the version history of the admin console after checking for updates, but the application version is not downloaded. The admin console temporarily displays an error message inline with this pending download version, informing the end user that they must update KOTS in order to download this application version. This error is also displayed when the end user checks for updates with the [`kots upstream upgrade` command](/kots-cli/upstream/upgrade/).
```

File: content/reference/v1beta1/application.md [hunk 2]
```diff
@@ -184,19 +184,13 @@ The minimum KOTS version that is required by the release. Introduced in KOTS v1.
...
 
-### Guidance for informing users of the need to update
+End users must update their admin console to the minimum KOTS version or later in order to download the version without error. KOTS cannot update itself automatically, and users cannot update KOTS from the admin console.
 
-After promoting a new release that specifies a minimum KOTS version that is later than what an end user currently has deployed, that release does not appear in the version history of the admin console after checking for updates. The admin console displays an error message temporarily, but it disappears after a few minutes. The admin console also displays this error when the user checks for updates with the [`kots upstream upgrade` command](/kots-cli/upstream/).
+After updating KOTS to an appropriate version, end users can return to the admin console and click the "Download" button next to the pending version in order to download it. You can also use the [`kots upstream download` command](/kots-cli/upstream/download) to do this. Once the version is downloaded, end users can proceed normally.
```
Comment (sgalsaleh) [inline content/reference/v1beta1/application.md:193]:
```suggestion
After updating KOTS to an appropriate version, end users can return to the admin console and click the "Download" button next to the pending download version in order to download it. You can also use the [`kots upstream download` command](/kots-cli/upstream/download) to do this. Once the version is downloaded, end users can proceed normally.
```

File: content/reference/v1beta1/application.md (hunk 1 above)
Comment (ajp-io) [reply content/reference/v1beta1/application.md:189]: You make a good point, though I agree that I'm not sure "pending download version" is perfect. Maybe I'll say, "The admin console temporarily displays an error message inline with this new version."

**Comment by ajp-io** [comment]
@replicatedhq/replicated-docs Ready for review. This is updated to reflect some enhancements we made this week.

**Review by paigecalvert — APPROVED** [review]

**Pull request merged**